sds/src/path.rs

```diff
@@ -52,8 +52,20 @@ impl<'a> Path<'a> {
         true
     }
 
+    fn size(&self) -> usize {
```

Comment:
If this is skipping non-Field segments, the function name should probably be more specific than `size()`. I would probably assume by just the function name that this is the number of segments in a path, not the sum of all of the bytes of fields.

Reply:
I've renamed the function to be `size_only_segments` 

[Rename size() in size_only_segments()](https://github.com/DataDog/dd-sensitive-data-scanner/pull/74/commits/b0e3c9ebcb37c2598615d5a346d2112e3b0c3ecf)